CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - diagnostic paths referring to `tests` folder
 
+### Cast
+
+#### Changed
+- bump starknet-rs dependency
+- replaced starknet-devnet with starknet-devnet-rs for tests
```

Comment:
Not sure if necessary in changelog

Reply:
A few people asked about it, so I figured I should include it. If you disagree, I can remove it no problem.

Reply:
We're generating releases' changelogs from this file so for me it makes sense to keep it focused on user-facing changes. Though I don't have a very strong opinion on that - if you think it's useful to include this change here we can do it.

Though I'd reword it so it's more obvious that you mean our own tests and not something related with testing, so it won't get confused with forge things.

Reply:
Ok makes sense actually. I have an opened comms channel with the people interested in this, so I can give them heads up this way, no reason to pollute the changelog. Deleted.